themes/custom/lex/templates/pages/page--calendar.html.twig

```diff
@@ -1,2 +1,94 @@
-{% extends 'page.html.twig' %}
 {% set content_attributes = content_attributes.addClass('lex-calendar-page') %}
```

Comment:
Really wondering if this can be condensed? Leave the page.html.twig in there and just overwrite a block. I don't see you needing to give this page all the blocks. Like where is the magic happening here?